EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -82,27 +85,80 @@ public struct MediaPlayerView: View {
             .wrapInButton(action: { viewModel.onTapAction?(control) })
     }
     
-    @ViewBuilder private func renderTimeElapsedWrapper<Content: View>(
+    @ViewBuilder private func renderTimePlaceholderWrapper<Content: View>(
         @ViewBuilder content: @escaping () -> Content
     ) -> some View {
         ZStack {
             content()
-            Text("10") // TODO: update with amount of rewind / forward
+            Text(String(format: Constants.seekFactorFormat, viewModel.seekFactor))
                 .foregroundColor(.white)
                 .font(.system(size: Constants.timeFontSize))
         }
     }
+
+    @ViewBuilder private func renderSeekTappableScreen() -> some View {
+        HStack(spacing: .zero) {
+            renderTappableScreen(control: .rewind, seek: .rewindTap)
+            renderTappableScreen(control: .forward, seek: .forwardTap)
+        }
+    }
+
+    @ViewBuilder private func renderTappableScreen(control: Control, seek: Seek) -> some View {
+        ZStack {
+            renderSeekTapsDetector(taps: Constants.doubleTap, control: control, seek: seek)
+            renderSeekTapsDetector(taps: Constants.singleTap, control: control, seek: seek)
+                .allowsHitTesting(viewModel.isTappingSeek)
+            renderSeekIcon(seek: seek)
+        }
+    }
+
+    @ViewBuilder func renderSeekTapsDetector(taps: Int, control: Control, seek: Seek) -> some View {
+        Rectangle()
+            .foregroundColor(.clear)
+            .contentShape(Rectangle())
+            .onTapGesture(count: taps) {
+                viewModel.onTapAction?(control)
+                viewModel.onSeekDoubleTap?()
+                renderSeekTaps(seek)
+            }
```

Comment:
I've never used `onTapGesture(count: Int)` before. What's the time preset on that? Does it even have a time leeway? I suspect that if I tap it, wait 10 seconds, and then tap it again it'll be triggered, won't it? We don't want that. I may be wrong though

Reply:
No time preset, this is handle by `.allowsHitTesting(viewModel.isTappingSeek)`. Sequential single taps are only available when this `viewModel.isTappingSeek` is true. This variable is toggled by `Controller` timer. 

`count` parameter is related to number of taps in the screen (default is 1 as normally use)